config/initializers/session_store.rb

```diff
@@ -3,8 +3,7 @@
 
 PrisonVisits2::Application.config.session_store :cookie_store,
   key: 'pvbs',
-  expire_after: 20.minutes,
+  expire_after: Rails.configuration.session_expires_after,
   httponly: true,
-  max_age: 20.minutes,
```

Comment:
We know that we don't need this because it has exactly the same function as `:expire_after`: Rails's [`CookieStore`](https://github.com/rails/rails/blob/3ac3760c69e6e6914c5ddae138856b3c82ac0f20/actionpack/lib/action_dispatch/middleware/session/cookie_store.rb#L60) is a subclass of [`Rack::Session::Abstract::ID`](https://github.com/rack/rack/blob/028438ffffd95ce1f6197d38c04fa5ea6a034a85/lib/rack/session/abstract/id.rb#L349-L350), in which they are treated identically.